hack/release.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${REPO_ROOT}/hack/util.sh"
+
+binary=$1
+os=$2
+arch=$3
+
+release_dir="${REPO_ROOT}/_output/release"
+mkdir -p "${release_dir}"
+
+tar_file="${binary}-${os}-${arch}.tgz"
+echo "!!! Packaging ${tar_file}"
+tar czf "${release_dir}/${tar_file}" "${REPO_ROOT}/LICENSE" -C "${REPO_ROOT}/_output/bin/${os}/${arch}" "${binary}"
+
+cd "${release_dir}"
+sha256sum "${tar_file}" > "${tar_file}.sha256"
```

Comment:
```suggestion
sha256sum "${tar_file}" > "${tar_file}.sha256-checksum.txt"
```
I'm not sure if it looks better. What do you think?

Reply:
apache also use `.sha*`

https://downloads.apache.org/tomcat/tomcat-10/v10.0.21/bin/apache-tomcat-10.0.21.zip.sha512
https://dlcdn.apache.org/httpd/httpd-2.4.53.tar.gz.sha256
https://www.apache.org/dist/kafka/1.0.0/kafka-1.0.0-src.tgz.sha512

Reply:
OK. Let's do it.
/lgtm
/approve